### PR TITLE
samtools@0.1: renamed samtools-0.1.rb

### PR DESCRIPTION
**Pull request by tmozgach**
### Have you:

- [x] Followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-science/blob/master/.github/CONTRIBUTING.md) document?
- [x] Checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-science/pulls) for the same update/change?
- [x] Run `brew audit --strict --online <formula>` (where `<formula>` is the name of the formula you're submitting) and corrected all errors?
- [x] Built your formula locally prior to submission with `brew install <formula>`?

### New formula: have you

- [ ] Written a sensible test? (The best test is to compile and run a sample program.)

### Updates to existing formula: have you

- [ ] Removed the `revision` line, if any, when bumping the version number?
- [ ] Added/bumped the `revision` number if the changes affect the precompiled bottles?
- [ ] Not altered the `bottle` section?
- [ ] Checked that the tests still pass?

### CODE REVIEW

File: formula_renames.json [hunk 1]
```diff
@@ -3,5 +3,6 @@
   "gal-sim": "galsim",
   "hdf@4": "hdf4",
   "snp_sites": "snp-sites",
-  "geant": "geant4"
+  "geant": "geant4",
+  "samtools-0.1": "samtools@0.1"
```
Comment (sjackman) [inline formula_renames.json]: Please sort this list alphabetically.

File: samtools@0.1.rb [hunk 2]
```diff
@@ -1,4 +1,5 @@
-class Samtools01 < Formula
+class SamtoolsAT01 < Formula
+  desc "Tools for manipulating next-generation sequencing data."
```
Comment (sjackman) [inline samtools@0.1.rb]: Please remove the period.

File: samtools@0.1.rb [hunk 3]
```diff
@@ -13,15 +14,9 @@ class Samtools01 < Formula
     sha256 "8238ae7e31e946ac98c259fccdecdcd929368257f4c174e8b17938a9f6e9d7a9" => :mountain_lion
     sha256 "f23f37c4afe69f7516cd3d1630ec50478e651bdc29ef5cefe1ee910d3575e308" => :x86_64_linux
   end
-
+  keg_only :versioned_formula
   option "with-dwgsim", "Build with 'Whole Genome Simulation'"
   option "without-bcftools", "Do not install BCFtools"
-
-  depends_on "ncurses" unless OS.mac?
```
Comment (sjackman) [inline samtools@0.1.rb]: Please keep this line.

File: samtools@0.1.rb [hunk 4]
```diff
@@ -13,15 +14,9 @@ class Samtools01 < Formula
     sha256 "8238ae7e31e946ac98c259fccdecdcd929368257f4c174e8b17938a9f6e9d7a9" => :mountain_lion
     sha256 "f23f37c4afe69f7516cd3d1630ec50478e651bdc29ef5cefe1ee910d3575e308" => :x86_64_linux
   end
-
+  keg_only :versioned_formula
```
Comment (sjackman) [inline samtools@0.1.rb]: Please add a blank line before and after this line.

**Comment by sjackman** [comment]
```
gcc -c -g -Wall  -O2 -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -D_USE_KNETFILE -D_CURSES_LIB=1 -DBGZF_CACHE -I. bgzf.c -o bgzf.o
In file included from bgzf.c:32:0:
bgzf.h:33:18: fatal error: zlib.h: No such file or directory
 #include <zlib.h>
```
https://travis-ci.org/Homebrew/homebrew-science/builds/235818937#L1203

Add…
```ruby
unless OS.mac?
  depends_on "ncurses"
  depends_on "zlib"
end
```

**Comment by sjackman** [comment]
`brew audit` fails due to
```
  * Dependency 'ncurses' may be unnecessary as it is provided by macOS; try to build this formula without it.
  * Dependency 'zlib' may be unnecessary as it is provided by macOS; try to build this formula without it.
==> FAILED
```
https://travis-ci.org/Homebrew/homebrew-science/builds/235856190#L1470
This audit failure only affects new formula, so I believe that we can ignore it.

File: cufflinks.rb [hunk 5]
```diff
@@ -1,4 +1,5 @@
 class Cufflinks < Formula
+  desc "Transcriptome assembly, differential expression analysis for RNA-Seq"
   homepage "http://cufflinks.cbcb.umd.edu/"
```
Comment (ilovezfs) [inline cufflinks.rb]: audit says this can use `https`

**Comment by tmozgach** [comment]
@ilovezfs  I changed it, thank you for the review.

**Comment by sjackman** [comment]
@ilovezfs Look good to merge?
Should the three commits related to renaming `samtools@0.1` be merged to a single commit?

**Comment by sjackman** [comment]
@ilovezfs Is bumping the bottle rebuild of `cufflinks` good enough, or should the `revision` be bumped?

**Comment by sjackman** [comment]
Hi, Tanya. Can you please rebase and squash these commits into two commits with the following commit messages:
```
samtools@0.1: Rename from samtools@0.1
cufflinks: Rename samtools-0.1 to samtools@0.1
```
I'll merge this PR after those changes.
The bottles built on Bintray last only 7 days if they're not published in that window, so this change will also have the desired side effect of rebuilding the bottles.

**Comment by tmozgach** [comment]
@sjackman  Done!

**Comment by sjackman** [comment]
Merged. Thanks, Tanya!

**Pull request closed without merging**